### PR TITLE
Add contribution guidelines and new bug report for v2

### PR DESCRIPTION
**Pull request by zdtsw**
## Description
ref: https://github.com/opendatahub-io/opendatahub-operator/issues/407

For the new issue template to work, need admin create new label "v1" "v2" first.

## Merge criteria:

- [x] The commits are squashed in a cohesive manner and have meaningful messages.
- [ ] Testing instructions have been added in the PR body (for PRs involving changes that are not immediately obvious).
- [ ] The developer has manually tested the changes and verified that the changes work

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -0,0 +1,20 @@
+# Contributing to the Opendatahub Operator
+
+Thanks for your interest in this project. You can contribute to this project in many different ways.
+
+## Issues and Enhancements
```
Comment (VaishnaviHire) [inline CONTRIBUTING.md:5]: I have added `component/<component-name>` labels to the repo. Can we add a line about adding appropriate labels when creating the issues.

Comment (zdtsw) [reply CONTRIBUTING.md:5]: updated.

**Comment by VaishnaviHire** [comment]
Shouldn't this PR be against feature-rearchitecture branch ?

**Comment by zdtsw** [comment]
> Shouldn't this PR be against feature-rearchitecture branch ?

will that pop out if merged to  a non-default branch for the new issue template? 
we can try first get it to feature-rearchitecture and see how it works

**Pull request merged**